Bug: Error parsing AmazonMQ message

### Expected Behaviour

AmazonMQ (ActiveMQ) message contains queue name in 
`...'destination': {'physicalName': 'MY.QUEUE.NAME'}...` as per [documentation](https://docs.aws.amazon.com/lambda/latest/dg/with-mq.html)
while parsing code is looking for 
`return self["destination"]["physicalname"]`
which causes parsing error

### Current Behaviour

Error when calling message.destination_physicalname

### Code snippet

```python
for message in event.messages:
        payload_decoded = message.decoded_data
        queue: str = message.destination_physicalname
```


### Possible Solution

[Fix typo here](https://github.com/aws-powertools/powertools-lambda-python/blob/a8cb3578e52634b95905c3cc612413dd985af33b/aws_lambda_powertools/utilities/data_classes/active_mq_event.py#L62)

### Steps to Reproduce

Send message via AmazonMQ, connect lambda function as event consumer

### Powertools for AWS Lambda (Python) version

latest

### AWS Lambda function runtime

3.11

### Packaging format used

Lambda Layers

### Debugging logs

```python
[ERROR] KeyError: 'physicalname'
Traceback (most recent call last):
  File "/var/task/archive.py", line 129, in lambda_handler
    queue: str = message.destination_physicalname
  File "/opt/python/aws_lambda_powertools/utilities/data_classes/active_mq_event.py", line 62, in destination_physicalname
    return self["destination"]["physicalname"]
```

--- a/aws_lambda_powertools/utilities/data_classes/active_mq_event.py
+++ b/aws_lambda_powertools/utilities/data_classes/active_mq_event.py
@@ -59,7 +59,7 @@ class ActiveMQMessage(DictWrapper):
 
     @property
     def destination_physicalname(self) -> str:
-        return self["destination"]["physicalname"]
+        return self["destination"]["physicalName"]
 
     @property
     def delivery_mode(self) -> Optional[int]:

--- a/tests/events/activeMQEvent.json
+++ b/tests/events/activeMQEvent.json
@@ -9,7 +9,7 @@
       "connectionId": "myJMSCoID",
       "redelivered": false,
       "destination": {
-        "physicalname": "testQueue"
+        "physicalName": "testQueue"
       },
       "timestamp": 1598827811958,
       "brokerInTime": 1598827811958,
@@ -25,7 +25,7 @@
       "connectionId": "myJMSCoID2",
       "redelivered": false,
       "destination": {
-        "physicalname": "testQueue"
+        "physicalName": "testQueue"
       },
       "timestamp": 1598827811958,
       "brokerInTime": 1598827811958,
@@ -42,7 +42,7 @@
       "connectionId": "myJMSCoID1",
       "persistent": false,
       "destination": {
-        "physicalname": "testQueue"
+        "physicalName": "testQueue"
       },
       "timestamp": 1598827811958,
       "brokerInTime": 1598827811958,

--- a/tests/unit/data_classes/test_active_mq_event.py
+++ b/tests/unit/data_classes/test_active_mq_event.py
@@ -27,7 +27,7 @@ def test_active_mq_event():
     assert message.broker_in_time == raw_event["messages"][0]["brokerInTime"]
     assert message.broker_out_time == raw_event["messages"][0]["brokerOutTime"]
     assert message.properties.get("testKey") == raw_event["messages"][0]["properties"]["testKey"]
-    assert message.destination_physicalname == raw_event["messages"][0]["destination"]["physicalname"]
+    assert message.destination_physicalname == raw_event["messages"][0]["destination"]["physicalName"]
     assert message.delivery_mode is None
     assert message.correlation_id is None
     assert message.reply_to is None

